teams/vsp/teams/insights-analytics/ga-events-data-dictionaries.md

```diff
@@ -742,116 +754,196 @@ _When the orientation of the way search results are presented is changed, either
+| Component Library Version | 133 | Hit | **No data is being collected**. |
 
 [Back to Table of Contents](#Data-Dictionaries)
 
 ## Custom Metrics
 
-Index|Name|Scope|Formatting Type|Active|
---- |:--- |:--- |:--- |--- |
-1|Event Start|Hit|Time|false|
-2|Event Complete|Hit|Time|false|
-3|IDme Login Attempt|Hit|Integer|true|
-4|IDme Login Success|Hit|Integer|true|
-5|MHV Login Attempt|Hit|Integer|true|
-6|MHV Login Success|Hit|Integer|true|
-7|DSLogon Attempt|Hit|Integer|true|
-8|DSLogon Success|Hit|Integer|true|
-9|Education - Count of Programs Saved|Hit|Integer|true|
-10|File Uploads|Hit|Integer|true|
-11|Disability Calculator Rows|Hit|Integer|true|
-12|Unused|Hit|Integer|false|
-13|Unused|Hit|Integer|false|
-14|Unused|Hit|Integer|false|
-15|Unused|Hit|Integer|false|
-16|Unused|Hit|Integer|false|
-17|GIBCT - Profile Benefit Estimate|Hit|Currency (Decimal)|true|
-18|Unused|Hit|Integer|false|
-19|Address Validation Needed|Hit|Integer|true|
-20|Address Suggestion Provided|Hit|Integer|true|
-21|Address Suggestion Used|Hit|Integer|true|
-22|Home Address Suggestion Failures|Hit|Integer|true|
-23|Mailing Address Suggestion Failures|Hit|Integer|true|
-24|Address Override|Hit|Integer|true|
+| Custom Metric Name | Index | Scope | Formatting Typpe | Definition |
+| --- | --- | --- | --- | --- |
+| Event Start | 1 | Hit | Time |  |
+| Event Complete | 2 | Hit | Time |  |
+| IDme Login Attempt | 3 | Hit | Integer | When a user attempts to login via IDme |
+| IDme Login Success | 4 | Hit | Integer | When a user successfully logs in via IDme |
+| MHV Login Attempt | 5 | Hit | Integer | When a user attempts to login via MHV |
+| MHV Login Success | 6 | Hit | Integer | When a user successfully logs in via MHV |
+| DSLogon Attempt | 7 | Hit | Integer | When a user attempts to login via DSLogon |
+| DSLogon Success | 8 | Hit | Integer | When a user successfully logs in via DSLogon |
+| Education - Count of Programs Saved | 9 | Hit | Integer | Occurs with a Vet Tec Completion with the edu-0994 form |
+| File Uploads | 10 | Hit | Integer | When a user uploads a file |
+| Disability Calculator Rows | 11 | Hit | Integer | Number of rows being used in the disability rating calculator found on /disability/about-disability-ratings/ |
+| Unused | 12 | Hit | Integer | - |
+| Unused | 13 | Hit | Integer | - |
+| Unused | 14 | Hit | Integer | - |
+| Unused | 15 | Hit | Integer | - |
+| Unused | 16 | Hit | Integer | - |
+| GIBCT - Profile Benefit Estimate | 17 | Hit | Currency (Decimal) |  |
+| Unused | 18 | Hit | Integer | - |
+| Address Validation Needed | 19 | Hit | Integer | This is triggered with a profile navigation action |
+| Address Suggestion Provided | 20 | Hit | Integer | This is triggered with a profile navigation action |
```

Comment:
_I kind of hate I know some of these things just by reading them_ might have done this work 😄 

```suggestion
| Address Suggestion Provided | 20 | Hit | Integer | Increments each time at least 1 address suggestion is provided at address validation |
```